### ISSUE TITLE
[Bug]: sp-picker does not read current item with NVDA or JAWS

### ISSUE THREAD

**Issue by jnurthen** [adobe/spectrum-web-components#1767]
### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Impacted component(s)

sp-picker

### Expected behavior

The current item in the listbox should be read out when navigating up/down in the listbox when using a screen reader

### Actual behavior

The currently selected item is (sometimes) read out when focus moves to the picker, but up and down arrow (while moving the focus visually) doesn't read the current item out

### What browsers are you seeing the problem in?

Firefox, Chrome, Microsoft Edge

### How can we reproduce this issue?

1. Go to https://opensource.adobe.com/spectrum-web-components/components/picker
2. Start NVDA
3. Open Selection Type picker
4. Picker opens and reads "list"
5. Up and Down arrows move focus but read nothing

### Logs taken while reproducing problem

It seems that focus is on the listbox not the option children.
2 possible resolutions:
* move focus to the option itself
* add aria-activedescendent to the listbox to point to the option

**Comment by jnurthen** [comment]
I've a feeling this might be being caused by https://github.com/adobe/spectrum-web-components/pull/1425

**Comment by Westbrook** [comment]
Yes, some parts of this stem from the work in #1425.

Reviewed via https://assistivlabs.com/ in NVDA and the proposed solution to #1766 does not directly address this situation. That update, in conjunction with the Combobox delivery in https://combobox-practicum-2--spectrum-web-components.netlify.app/storybook/?path=/story/combobox--kerning (this preview currently features both changes) does seem to address the situation in that context.

I'll review the delta there and see if there is a simple way to support this in the current Picker approach. Otherwise, being the current target its to port back the combobox structure as read-only to the Picker anyway, the best path may be further prioritizing that work.

**Comment by jnurthen** [comment]
I get the same issues with https://combobox-practicum-2--spectrum-web-components.netlify.app/storybook/?path=/story/combobox--kerning too. Indeed I can reproduce this in the simple menu components - https://opensource.adobe.com/spectrum-web-components/components/menu
